CHANGELOG.md

```diff
@@ -9,6 +9,9 @@
 
 - Support filter with `tag` attribute used by AWS on `aws_autoscaling_group` resource
   ([Issue #223](https://github.com/cycloidio/terracognita/issues/223))
+- Resource name improved by low casing all the options
```

Comment:
It's not all options but tags only I believe

Reply:
Well, all options we provide to get the name which is (in this order):
* Tag.Name
* resource.ID
* Random alphanumeric of len == 5